packages/components/Button/src/Button.types.ts

```diff
@@ -81,7 +82,7 @@ export interface IButtonTokens extends FontTokens, IForegroundColorTokens, IBack
   /**
    * Source URL or name of the icon to show on the Button.
    */
-  icon?: string;
+  icon?: number | string | IconProps;
```

Comment:
Does this really need to be in tokens? I guess the use case is setting a default icon value but it does seem a bit strange. (I realize I may have put it there initially)